debian/control

```diff
@@ -15,7 +15,7 @@ Build-Depends: bash-completion,
                dh-exec,
                dh-golang,
                git,
-               golang-1.13-go,
+               golang-go,
```

Comment:
Does this make it harder to backport?  (IIRC, Docker 20.10.8+ or maybe 9+ requires at least Go 1.16+)

Reply:
It does indeed, [this version depends on Golang 1.16.12](https://github.com/moby/moby/releases/tag/v20.10.12) which does not build even with Golang 1.16 in Ubuntu (in Jammy and Impish we have 1.17 as the default). So to backport this version to Focal and Bionic we will need to find another solution, I'll be discussing with others if we can try to backport in some way Golang 1.17 to Focal and Bionic. Unfortunately we are unable to keep using an old Golang version.

Reply:
Sorry, it builds with Golang 1.16 in the archive but it is not available in Jammy (it was removed). So this version can be backported to Focal but not to Bionic where we do not have Golang >= 1.16.